src/firewall/varktables/types.rs

```diff
@@ -200,44 +203,96 @@ pub fn get_network_chains(
     ));
     chains.push(postrouting_chain);
     if !is_ipv6 {
-        // NETAVARK_FORWARD
-        let mut netavark_forward_chain =
-            VarkChain::new(conn, FILTER.to_string(), NETAVARK_FORWARD.to_string(), None);
-        netavark_forward_chain.create = true;
-
-        // Create incoming traffic rule
-        // CNI did this by IP address, this is implemented per subnet
-        netavark_forward_chain.build_rule(VarkRule::new(
-            format!(
-                "-d {} -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT",
-                network
-            ),
-            Some(TeardownPolicy::OnComplete),
-        ));
-
-        // Create outgoing traffic rule
-        // CNI did this by IP address, this is implemented per subnet
-        netavark_forward_chain.build_rule(VarkRule::new(
-            format!("-s {} -j ACCEPT", network),
-            Some(TeardownPolicy::OnComplete),
-        ));
-        chains.push(netavark_forward_chain);
-
         // FORWARD chain
-        // Insert the rule into the first position
         let mut forward_chain = VarkChain::new(conn, FILTER.to_string(), FORWARD.to_string(), None);
+        add_chain_unique(
+            forward_chain.driver,
+            &forward_chain.table,
+            &forward_chain.chain_name,
+        )?;
+
+        // used to prepend specific rules
+        let mut ind = 1;
+
+        debug!("ISOLATION: {}", isolation);
```

Comment:
This debug log is leaving no context, it would be hard to figure out what does this means later.